src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetrySerializer.cs

```diff
@@ -8,8 +8,9 @@
 
     internal class TelemetrySerializer
     {
+        internal readonly Transmitter Transmitter;
```

Comment:
I'll be happy to follow any other ruleset. Keeping it compliant to the current ruleset for now.